compiler/rustc_target/src/spec/i686_unknown_hurd_gnu.rs

```diff
@@ -0,0 +1,21 @@
+use crate::spec::{Cc, LinkerFlavor, Lld, StackProbeType, Target};
+
+pub fn target() -> Target {
+    let mut base = super::hurd_gnu_base::opts();
+    base.cpu = "pentium4".into();
```

Comment:
I wonder if this is copied from another target? We have received great discontent from people of various distros regarding the use of `"pentium4"` as the base CPU for so-called "i686" targets. Of course, people from other distros think it's fine to use the Pentium 4 for a tuple that starts with `i686-`. Or `i586-`. Or even `i386-`! The last is a somewhat notorious choice by an obscure operating system derived in part from the Berkeley Software Distribution 4.2, but you probably haven't heard of it, I'm sure: the "Darwin" OS?

Anyways, to be clear, this is perfectly acceptable if it's on purpose, and for Linux at least, we have officially committed to this decision by now. But part of our rationale was that preferably this string should match whatever LLVM thinks the default CPU is for this target tuple. That way, people compiling for this target can fully benefit from cross-language interprocedural optimizations when they have properly configured things for a Rust + Clang project that correctly shares an LLVM.

Reply:
We essentially copied this over from almost all the other i686 targets. Yes we do know about Darwin :) it's using the Mach microkernel too.

We can indeed rather stick to pentium pro, to accommodate for older hardware, done so.